app/models/claim/base_claim.rb

```diff
@@ -349,9 +349,21 @@ def perform_validation?
       force_validation? || validation_required?
     end
 
-    # we must validate unless it is being created as draft from any source except API or is in state of archive_pending_delete or deleted
+    # validation required when
+    # being created from api (as draft)
+    # or is in state of archived_pending_delete or draft (not from api)
+    # or is in a state deleted (old statement????)
+    # or is transitioning via certain events
+    #
     def validation_required?
-      from_api? || !(draft? || archived_pending_delete?)
+      return true if from_api?
+      return false if draft? || archived_pending_delete?
```

Comment:
Could this be
``` ruby
return false if draft? || archived_pending_delete? || disabled_for_transition?
```
Or was it separated for a reason?

Reply:
purely seaprate as the disabled for transition was my main focus - I can't see any reason not to  make it as you suggest, will try!